casper/src/main/scala/coop/rchain/casper/BlockMetadata.scala

```diff
@@ -0,0 +1,22 @@
+package coop.rchain.casper
+
+import com.google.protobuf.ByteString
+
+import coop.rchain.casper.Estimator.BlockHash
+import coop.rchain.casper.protocol.{BlockMessage, Justification}
+
+final case class BlockMetadata(blockHash: BlockHash,
+                               parents: List[BlockHash],
+                               sender: ByteString,
+                               justifications: List[Justification])
```

Comment:
I assume we will use this class as "BlockMessage without Body" you mentioned previously. Don't you think we need other fields from `BlockMessage` and `Header` here?

Reply:
Indeed, that is what I was going for here. I think the fields there are sufficient. But if we realize that we need more then it is easy to add them to this case class.